### DISCUSSION TITLE
[Feature] Create specific-sized box

### DISCUSSION THREAD

**Discussion by alicera** [wkentaro/labelme#1065] [Ideas / Feature requests]
I want to create a box with (256x256).
How can I do with labelme?

**Comment by wkentaro** [comment]
I'm not sure if there are many needs for this.

**Comment by alicera** [comment]
I have a idea.
Assume the label is 1 class person.
I label many images and I find some images have the ambiguous status.
The ambiguous status define fully black person, half person, and occlude etc.
So I want to create a [specific-sized box](https://www.tomshardware.com/reviews/what-is-fhd-full-hd,5741.html) to cut it. And it can avoid the ambiguous status.

It is inspired 
https://github.com/ultralytics/yolov5/issues/1423

**Comment by wkentaro** [comment]
What do you mean by “cut it”?

**Comment by alicera** [comment]
REF ：https://github.com/biubug6/Pytorch_Retinaface

The blue circle is a ambiguous status. If I label it, it maybe affect the detect result. 
![image](https://user-images.githubusercontent.com/59388989/156131522-81e521d2-05e2-4cf8-8c3e-934886565f83.png)

So I can crop it. And label it
![image](https://user-images.githubusercontent.com/59388989/156132384-07fe9e5a-f28d-4a02-a3ad-6d398d3c0587.png)

**Comment by wkentaro** [comment]
So you mean you want to "ignore it" by "cut it"?
Why not just use `--labelflags`?
https://github.com/wkentaro/labelme/tree/main/examples/instance_segmentation#annotation

```
labelme data_annotated --labels labels.txt --nodata --labelflags '{.*: [occluded, truncated], person: [male]}'
```

**Comment by alicera** [comment]
I dont want to write the info [occluded] into the json.And it will contain the original image not the crop. It is not the suitable for me.
The crop have another advantage.For example, my data is 4:3(w:h), but I search some 16:9 data. I can crop it with 4:3. Let the data contain 4:3 ratio.
